app/src/main/java/by/academy/jc/zhabko/MyClass.java

```diff
@@ -0,0 +1,24 @@
+
+package by.academy.jc.zhabko;
+
+public class MyClass {
+
+  public static void main(String[] args) {
+
+    // Basic example
+    System.out.println(new MyClass().getHW());
+
+    // Defining new variable "message"
+    String message = "Second example";
+
+    // Sending message to the terminal
+    System.out.println(message);
+
+    // To see the result you have to use "javac command" and terminal
+    for (String arg : args) {
+      System.out.println(arg);
+    }
+  }
+  public String getHW() {return "Hello World!";}
```

Comment:
`HW` is not common abbreviation like `ID` or `IBAN`. Please use verbose name in here.

Having `get` in the beginning could tell another developer that this class should have `HW` property and this is a getter. If it's not, please use another name